src/frontend/src/pages/PIAIntakeForm/index.tsx

```diff
@@ -515,6 +532,28 @@ const PIAFormPage = () => {
                 </div>
               </div>
             )}
+            {!pathname.includes('intake') && (
```

Comment:
Do we want to add an explicit "link" to the logic? What if the 1st page changes to something else, will this break?

Reply:
yes, it does break the logic if the page/tabs change , if the first tab name is not intake anymore, then this will not work